shared-utils/verify_preflight.sh

```diff
@@ -88,9 +88,15 @@ fi
 
 echo ">>>> Verify oc get nodes"
 echo ">>>>>>>>>>>>>>>>>>>>>>>>>"
-if [[ $(oc get nodes | grep -i ready | wc -l) -ne 1 ]] && [[ $(oc get nodes | grep -i ready | wc -l) -ne 3 ]]; then
+# TO-DO: Installed Vshpere with assisted installer and enable vshpere during deployment
```

Comment:
don't understand this comment TO-DO, why do you need to install vsphere with AI?

Reply:
Right one my vSphere cluster has 3 masters and 3 workers so it is failing on the check. I wanted to add the vShpere check because I know that baremetal can be sno or 3 nodes.  vShpere can have more that 3 worker nodes in some environments.